src/adapter.js

```diff
@@ -59,6 +60,84 @@ function useResourceGroups(api) {
   return tags.length > 0;
 }
 
+// Look up a position in the original source based on a JSON path, for
+// example 'paths./test.get.responses.200'
+function getPosition(ast, path) {
```

Comment:
If it is true [JSONPath](http://goessner.net/articles/JsonPath/), you should be able to wrap keys in brackets.

```
x.store.book[0].title
```

is the same as:

```
x['store']['book'][0]['title']
```